utils/sanitize_dashboards.ts

```diff
@@ -5,6 +5,7 @@ import * as glob from 'glob';
 const sanitizeRawContent = (rawConfig: string): string => {
   return rawConfig
     .replace(/\"accountId\"\s*:\s*\d+/g, '"accountId": 0') // Anonymize account ID
+    .replace(/\"accountIds\"\s*:\s*\d+/g, '"accountIds": []')
```

Comment:
I think we would want to capture anything that comes after `accountIds`. After reviewing some of the prometheus quickstarts we had some different use cases.

Examples:
- `accountIds: [ 0 ]`
- `accountIds: [ 524532, 543223 ]`

Playing around with regex I came up with this:
```suggestion
    .replace(/\"accountIds\"\s*:\s.*/g, '"accountIds": []')
```
<img width="729" alt="Screen Shot 2022-11-28 at 2 55 16 PM" src="https://user-images.githubusercontent.com/56952981/204398082-cae0f512-7165-4117-a276-b3781dc3df35.png">

This will capture anything that comes after `accoundIds`